src/components/experiment-tracking/run-export-modal.js/run-export-modal.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import RunExportModal from './index';
+import Adapter from 'enzyme-adapter-react-16';
+import { configure, mount, shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+
+// to help find text which is made by multiple HTML elements
```

Comment:
Is there a place I can put this util function? I think we can re-use it somewhere else later

Reply:
i was thinking maybe in  src/components/ui/button we could add a success button?

Reply:
yeah i was thinking the same, what do you think of naming it `button-success`. It sounds weird but if we name it this way, all the button components display next to each other.
eg 
```
- button
- button-success
```

what do you think @rashidakanchwala @tynandebold ?

Reply:
We should probably generalize it more than that. At some point we'll probably need to supply a `fail` state, too. So maybe we do something like `button-message` or similar.

Also to your original question about the utility function: you could just `export const findTextWithTags` for now, because at the moment I don't think we need it anywhere else, so it may be premature to move it elsewhere. If you want to move it though, somewhere in one of the [utils files](https://github.com/kedro-org/kedro-viz/tree/main/src/utils) would be best, but you'd probably want to create a new file, like `test-utils.js`, so maybe we hold off for now.